scaladoc/src/dotty/tools/scaladoc/ScaladocSettings.scala

```diff
@@ -93,6 +93,12 @@ class ScaladocSettings extends SettingGroup with AllScalaSettings:
     "./docs"
   )
 
+  val noLinkWarnings: Setting[Boolean] = BooleanSetting(
+    "-no-link-warnings",
+    "Avoid warnings for ambiguous and incorrect links ONLY in members look up. Doesn't affect incorrect links of assets etc.",
```

Comment:
I would change the message since now (because of the ONLY) it can be understood that: if the flag is set to false then all warnings are suppressed.

Reply:
Fixed